src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -866,4 +942,46 @@ public BuildRule getBinaryBuildRule() {
     }
     return ImmutableSortedSet.of();
   }
+
+
+  @Override
+  public Iterable<? extends NativeLinkable> getNativeLinkableDeps(CxxPlatform cxxPlatform) {
+    return ImmutableSet.of();
+  }
+
+  @Override
+  public Iterable<? extends NativeLinkable> getNativeLinkableExportedDeps(CxxPlatform cxxPlatform) {
+    return ImmutableSet.of();
+  }
+
+  @Override
+  public NativeLinkableInput getNativeLinkableInput(
+      CxxPlatform cxxPlatform, Linker.LinkableDepType type) throws NoSuchBuildTargetException {
+    ImmutableList.Builder<Arg> linkerArgsBuilder = ImmutableList.builder();
+    linkerArgsBuilder.addAll(StringArg.from(
```

Comment:
Fixed this with no linker flags and an `install_name_tool` step in the apple bundle build rule.